libvirt-glib.rb

```diff
@@ -0,0 +1,105 @@
+class LibvirtGlib < Formula
+  desc "GLib and GObject mapping for libvirt"
+  homepage "http://www.libvirt.org"
+  url "http://libvirt.org/sources/glib/libvirt-glib-0.2.3.tar.gz"
+  sha256 "c3b11aa34584b3416148b36bb005505e461830c73c9fd2f40143cca3309250dd"
+
+  depends_on "gettext" => :build
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+
```

Comment:
Yes, they (gtk-vnc, libvirt-glib, libosinfo) are all authored by me. Will address this later